### ISSUE TITLE
Build a portable stacks-node image

### ISSUE THREAD

**Issue by hugocaillard** [stacks-network/stacks-core#4557]
The Devnet (`clarinet devnet`) is used by developer to run a local Bitcoin node, Stacks node, Signer node, and more.
Hence we need to be able to run stacks docker image on any platform.

Currently, the devnet runs `linux/amd64` image, even on macos through docker emulation.  But it doesn't seem to work anymore on images built from source recently.

**Comment by wileyj** [comment]
to add some context, the images are no longer portable due to the inclusion of jemalloc to x64 builds. 

@BowTiedDevOps two things we can look at here:
1. multi-platform build - for each PR, could we build for both x64 and aarch64 (and how much longer would it take to do so)?
2. disable jemalloc from x64 PR image builds. this might be a little tricky, but i think it's doable

ref:
https://github.com/stacks-network/stacks-core/pull/4460
https://github.com/stacks-network/stacks-core/pull/4367

**Comment by wileyj** [comment]
on an aarch64 machine, running `docker run --platform linux/amd64 blockstack/stacks-blockchain:feat-2.4.x-nakamoto-migration stacks-node mainnet start` errors with:
```
<jemalloc>: MADV_DONTNEED does not work (memset will be used instead)
<jemalloc>: (This is the expected behaviour if you are running under QEMU)
qemu: uncaught target signal 11 (Segmentation fault) - core dumped
```

**Comment by smcclellan** [comment]
@BowTiedDevOps @wileyj Is this issue meant to be closed?

**Comment by wileyj** [comment]
yes - this was solved prior to 2.5
